scipy/stats/stats.py

```diff
@@ -3339,9 +3339,45 @@ def pointbiserialr(x, y):
     pvalue : float
         2-tailed p-value
 
+    Notes
+    -----
+    For testing the hypothesis that the point-biserial correlation coefficient
+    is significantly different from zero, a t-test with n-2 degrees of freedom
+    should be used. Pearsonr is equivalent.
+
+    The value of the point-biserial correlation can be calculated from:
+
+    .. math::
+
+        r_{pb} = \frac{\overline{Y_{1}} - \overline{Y_{0}}}{\overline{s_{y}}}\sqrt{\frac{N_{1} N_{2}}{N (N - 1))}}
+
+    Where ``:math:Y_{0}`` and ``:math:Y_{1}`` are means of the metric
+    observations coded 0 and 1 respectively; ``:math:N_{0}`` and ``:math:N_{1}``
+    are number of observations coded 0 and 1 respectively; ``:math:N`` is the
+    total number of observations and ``:math:\overline{s_{y}}`` is the standard
+    deviation of all the metric observations.
+
+    A value of ``:math:r_{pb}`` that is significantly different from zero is
+    completely equivalent to a significant difference in means between the two
+    groups. Thus, an independent groups t Test with ``:math:N-2`` degrees of
```

Comment:
Same in the rest of this section. Double backticks go around Python code snippets, not LaTeX.